contents/blog/using-posting.md

```diff
@@ -0,0 +1,252 @@
+
+### 1. Replace multiple services with just PostHog
+
+Let's start with a big one.
+
+You could use Amplitude for product analytics, Hotjar for session recording, Flagsmith for feature control, Segment as your customer data platform, Optimizely for testing, and Redshift for your data warehouse – these are sensible choices.
+
+But you can replace some (or even all) of them with just PostHog.
+
+We've built PostHog so companies don't have to run multiple services to get the insights they need. 
+
+[PostHog Cloud](/pricing) delivers a complete set a tools to help you build better products, while our integration with numerous data platforms means many of our customers end up ditching products like Segment for organizing and syncing customer data. 
+
+And if you [self-host PostHog](/docs/self-host), you get a built-in data warehouse based on ClickHouse into the bargain. 
+
+## 2. Build an AARRR pirate metrics dashboard
+
+![AARRR pirate metrics](../images/blog/things-to-do-in-posthog/aarrr.png)
+
+AARRR (Acquisition, Activation, Retention, Revenue, Referral) is a classic framework for driving product-led growth, and it's a great place to start when building your first dashboard in PostHog.
+
+What metrics you choose will depend on the business and it's fine to have more than one metric for each category. What's vital, however, is that they're easy to understand, actionable and comparable.
+
+Read our guide to [getting started with the AARRR framework](/tutorials/aarrr-framework) and [B2B product metrics](/blog/b2b-saas-product-metrics) guides for more.
+
+## 3. Discover who your power users are
+
+![discovering power users](../images/blog/things-to-do-in-posthog/cohorts.png)
+
+You can learn a huge amount by discovering who your most active users are, and what they're doing.
+
+What are the features they use the most? How often do they perform specific tasks? What job title or role do these people have in their company? 
+
+These are the kinds of insights that inform your marketing efforts, your product roadmap, and help you find product market fit.
+
+For this you'll want to employ our powerful Cohorts tool by using the "completed event regularly" or "completed an event multiple times" conditions. 
+
+An e-commerce platform might ask for "users who bought an item in 5 out of the last 7 weeks" – we track many things, including users who view dashboards frequently.
+
+## 4. Gather user feedback
+
+![pineapple on pizza](../images/blog/things-to-do-in-posthog/pinepapple.png)
+
+You can [use PostHog to gather feedback on your product](/tutorials/survey), website, or conduct user polls.
+
+On our website, every docs page has a question asking whether the page was useful – we use this to help identify pages that need improvement. We also ask anyone who gets a 404 on our website whether pineapple belongs on pizza. We use this to report people to the pizza police. 
```

Comment:
```suggestion
Because PostHog is all about tracking events, it's easy to create your own simple feedback interfaces and analyse the results in PostHog.

On our website, every docs page has a question asking whether the page was useful – we use this to help identify pages that need improvement. We also ask anyone who gets a 404 on our website whether pineapple belongs on pizza. We use this to report people to the pizza police. 

Read our [running surveys with no backend](/tutorials/survey) tutorial to learn how.
```